flink-table/flink-table-planner/src/test/java/org/apache/flink/table/catalog/CatalogManagerTest.java

```diff
@@ -102,19 +102,20 @@ public void testRegisterCatalogWithExistingName() throws Exception {
 	}
 
 	@Test
-	public void testReplaceTemporaryTable() throws Exception {
+	public void testIgnoreTemporaryTableExists() throws Exception {
 		ObjectIdentifier tempIdentifier = ObjectIdentifier.of(
 			BUILTIN_CATALOG_NAME,
 			BUILTIN_DEFAULT_DATABASE_NAME,
 			"temp");
 		CatalogManager manager = root()
 			.builtin(
 				database(BUILTIN_DEFAULT_DATABASE_NAME))
-			.temporaryTable(tempIdentifier)
 			.build();
 
 		CatalogTest.TestTable table = new CatalogTest.TestTable();
 		manager.createTemporaryTable(table, tempIdentifier, true);
+		CatalogTest.TestTable anotherTable = new CatalogTest.TestTable();
+		manager.createTemporaryTable(anotherTable, tempIdentifier, true);
```

Comment:
Minor: add test for false `ignore`?

Reply:
The next `testTemporaryTableExists` test the case "ignoreIfExists=false"